test/integration/analysis-controller-multiple-providers/install.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: AnalysisValueTemplate
+metadata:
+  name: value-1
+  namespace: $NAMESPACE
+spec:
+  provider:
+    name: my-first-mocked-provider
+    namespace: $NAMESPACE
+  query: 'query-1'
+---
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: AnalysisValueTemplate
+metadata:
+  name: value-2
+  namespace: $NAMESPACE
```

Comment:
Would it be possible to remove the hardcoded namespaces? Since in the current state the namespace should be fetched automatically when not present

Reply:
yep, just adapted to that change and removed the $NAMESPACE wherever possible